src/JustSaying/Extensions/JsonSerializerOptionsExtensions.cs

```diff
@@ -9,18 +9,9 @@ internal static class JsonSerializerOptionsExtensions
 {
     public static JsonTypeInfo<T> GetTypeInfo<T>(this JsonSerializerOptions options)
     {
-        foreach (var info in options.TypeInfoResolverChain)
-        {
-            Console.WriteLine(info);
-        }
-
+        // This is not guarded as we want to throw if the desired type has not been configured for
```

Comment:
What does it throw though? We wouldn't want, say, a `NullReferenceException` trying to do the cast.

Reply:
I'll update the comment, it's an argument exception if JsonSerializerOptions can't resolve a JsonTypeInfo for T. I think it'll be more understandable as an error if we just let the default error propagate, and this follows the pattern used by request delegate generators to access the typed type info (Although they know the value of T at compile time)